app/ledger/neonLedger.js

```diff
@@ -76,8 +77,11 @@ export default class NeonLedger {
     const paths = await NeonLedger.list()
     if (paths.length === 0) throw new Error(MESSAGES.NOT_CONNECTED)
     if (paths[0]) {
-      const ledger = new NeonLedger(paths[0])
-      return ledger.open()
+      let ledger = new NeonLedger(paths[0])
```

Comment:
If we are not reassigning `ledger` why is it being instantiated with `let`?

Reply:
resolved in d62e32b50d6708e2f72e8b6000ea81f659058a15